### PR TITLE
Spelling fixes

### PR DESCRIPTION
**Pull request by scop**

### CODE REVIEW

**Comment by trapas** [comment]
Can one of the admins verify this patch?

File: src/lib/notify_new_path.c [hunk 1]
```diff
@@ -21,10 +21,10 @@
 void notify_new_path(const char *path)
 {
     /* Ignore results and don't wait for response -> NULL */
-    notify_new_path_with_reponse(path, NULL);
+    notify_new_path_with_response(path, NULL);
 }
 
-int notify_new_path_with_reponse(const char *path, char **message)
+int notify_new_path_with_response(const char *path, char **message)
```
Comment (xsuchy) [inline src/lib/notify_new_path.c]: Can you please alter https://github.com/abrt/abrt/blob/master/src/dbus/abrt_problems2_task_new_problem.c#L170 too?

Comment (scop) [inline src/lib/notify_new_path.c]: Done.

**Comment by xsuchy** [comment]
test this please

File: src/lib/notify_new_path.c (hunk 1 above)
Comment (xsuchy) [inline src/lib/notify_new_path.c]: Thank you

**Comment by xsuchy** [comment]
The failure is not relevant to this PR, so merging.

**Pull request merged**